php/WP_CLI/Runner.php

```diff
@@ -103,6 +103,12 @@ public function get_global_config_path() {
 		} else {
 			$config_path                    = Utils\get_home_dir() . '/.wp-cli/config.yml';
 			$this->global_config_path_debug = 'Using default global config: ' . $config_path;
+
+			// If global config doesn't exist create one.
+			if ( ! file_exists( $config_path ) ) {
+				$this->global_config_path_debug = "Default global config doesn't exist, creating one in {$config_path}";
+				Process::create( Utils\esc_cmd( 'touch %s', $config_path ) )->run();
+			}
 		}
```

Comment:
This should run for both cases, when `WP_CLI_CONFIG_PATH` is set, as well as when we default to the home folder.

```suggestion
		}

		// If global config doesn't exist create one.
		if ( ! file_exists( $config_path ) ) {
			$this->global_config_path_debug = "Default global config doesn't exist, creating one in {$config_path}";
			Process::create( Utils\esc_cmd( 'touch %s', $config_path ) )->run();
		}
```